contributors/design-proposals/node/seccomp.md

```diff
@@ -191,7 +197,8 @@ profiles to be opaque to kubernetes for now.
 
 The following format is scoped as follows:
 
-1.  `docker/default` - the default profile for the container runtime
+1.  `docker/default` - the default profile for the container runtime, can be
```

Comment:
this is the default for all runtimes? or just the docker runtime?

Reply:
Yeah, should be `runtime/default` or `container/default` or `default`.

Reply:
I agree that we should use a general term to cover all kinds of runtimes. The current immediate step is to enable docker/default for docker runtime, which covers a lot of our use cases. Having a general seccomp default will need spec, ect.